app/src/main/java/org/mozilla/focus/web/ClassicWebViewProvider.java

```diff
@@ -142,20 +145,20 @@ public static void disableBlocking(WebSettings settings, WebView webView) {
         CookieManager.getInstance().setAcceptCookie(true);
     }
 
-    public static void requestDesktopSite(WebSettings settings) {
+    public void requestDesktopSite(WebSettings settings) {
         settings.setUserAgentString(toggleDesktopUA(settings, true));
         settings.setUseWideViewPort(true);
     }
 
-    public static void requestMobileSite(Context context, WebSettings settings) {
+    public void requestMobileSite(Context context, WebSettings settings) {
         settings.setUserAgentString(toggleDesktopUA(settings, false));
         settings.setUseWideViewPort(false);
     }
 
     /**
      * Build the browser specific portion of the UA String, based on the webview's existing UA String.
      */
-    @VisibleForTesting static String getUABrowserString(final String existingUAString, final String focusToken) {
+    public String getUABrowserString(final String existingUAString, final String focusToken) {
```

Comment:
\[PMD\] INFO: The method 'getUABrowserString\(String, String\)' has a NCSS line count of 12\.